### PR TITLE
Complete modal-dialog code for task-02

### PR DESCRIPTION
**Pull request by Marcinw-455**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-jakieś cos
\ No newline at end of file
+references  
+window  
+document    
+document.body      
+element
+document.getElementById
```
Comment (MateuszKuriata) [inline README.md:6]: this is not necessary there;

File: dist/index.html [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
-    <link rel="stylesheet" href="styles/main.css">
 </head>
 <body>
+<div id="ida"></div>
     <div class="main-header">
         <span>Works</span>
     </div>
+    <div id="main-container"></div>
 <button id="dialog-button">Test</button>
+
```
Comment (MateuszKuriata) [inline dist/index.html:16]: fix indents and white new lines

File: src/js/dialog.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    const element = document.createElement(type);
+    element.classList = classList;
+    return element;
+}
+
+const createDialogWindow = function () {
+    const modalDialog = createDialogElement("div", "modal-dialog")
+    document.getElementById("main-container").appendChild(modalDialog);
+    const modalContent = createDialogElement("div", "modal-content");
+    document.querySelector(".modal-dialog").appendChild(modalContent);
```
Comment (MateuszKuriata) [inline src/js/dialog.js:33]: you can refer to this element by variable instead of querying this element from DOM

File: src/js/dialog.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    const modalHeader = createDialogElement("div", "modal-header");
+    document.querySelector(".modal-content").appendChild(modalHeader);
```
Comment (MateuszKuriata) [inline src/js/dialog.js:35]: same here, and all querySelectors below

File: src/js/dialog.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    document.querySelector(".modal-content").appendChild(modalFooter);
+    const modalFooterButtonClose = createDialogElement("button", "modal-footer-button-close");
+    document.querySelector(".modal-footer").appendChild(modalFooterButtonClose);
+    document.querySelector(".modal-footer-button-close").innerHTML = "close";
+    document.querySelector(".modal-footer-button-close").setAttribute("data-dismiss", "modal");
+    const modalFooterButtonSave = createDialogElement("button", "modal-footer-button-save");
+    document.querySelector(".modal-footer").appendChild(modalFooterButtonSave);
+    document.querySelector(".modal-footer-button-save").innerHTML = "Save changes";
+}
+
```
Comment (MateuszKuriata) [inline src/js/dialog.js:56]: please remove empty lines

File: src/js/dialog.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    const dialogButton = document.getElementById('dialog-button');
+    dialogButton.onclick = function () {
+        createDialogWindow();
+        saveButton();
+        closeButtons();
+    };
+}
+    const saveButton = function () {
+        document.querySelector(".modal-footer-button-save").addEventListener('click', function () {
+            alert("Sukces");
```
Comment (MateuszKuriata) [inline src/js/dialog.js:12]: typo

File: src/index.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import initDialog from './js/dialog.js';
+console.log("hello world");
+console.log("hello world2");
+console.log("hello world2");
+console.log("hello world2");
+initDialog();
+import createDialogWindow from "./js/dialog.js";
+createDialogWindow();
+import createDialogElement from "./js/dialog.js";
```
Comment (MateuszKuriata) [inline src/index.js:9]: please fix imports and code execution

**Review by MateuszKuriata — CHANGES_REQUESTED** [review]

**Comment by Marcinw-455** [comment]
the code has been corrected

**Comment by Marcinw-455** [comment]
code and structure is improved

**Review by MateuszKuriata — APPROVED** [review]

**Pull request merged**